test/test_jit_fuser_te.py

```diff
@@ -123,6 +123,27 @@ def func(x):
         scripted = self.checkScript(func, (a,))
         self.assertLastGraphAllFused()
 
+    @unittest.skipIf(IS_SANDCASTLE, "NYI: fuser CPU support for Sandcastle")
```

Comment:
I guess we need to import `IS_SANDCASTLE` from `torch.testing._internal.common_utils`.